assets/js/dintero-checkout-express.js

```diff
@@ -320,7 +320,7 @@ jQuery( function( $ ) {
 					}
 
 					if ( ! $( '#billing_last_name' ).val().trim() ) {
-						$( '#billing_last_name' ).val( 'N/A' );
+						$( '#billing_last_name' ).val( '⠀' );
```

Comment:
Setting an empty value could have the same effect in some checks as not setting it at all. Is there a real reason for this change or is it just that it looks better? Because if something goes wrong with an order this would make it harder to debug

Reply:
It is mostly for appearance, but I can revert the change. This is a "blank braille pattern" while it appears as empty space, it is a valid non-empty character.